packages/headless/src/features/configuration/configuration-actions-loader.ts

```diff
@@ -100,5 +111,6 @@ export function loadConfigurationActions(
     setOriginLevel3,
     updateAnalyticsConfiguration,
     updateBasicConfiguration,
+    setAnonymous,
```

Comment:
Is a dedicated action needed? Or can we leverage the `updateAnalyticsConfiguration` action?

Reply:
Answered below.